FlowCrypt/Functionality/Mail Provider/Message Provider/MessageService.swift

```diff
@@ -215,12 +217,14 @@ final class MessageService {
             let err = decryptErrBlock.decryptErr?.error
             text = "Could not decrypt:\n\(err?.type.rawValue ?? "UNKNOWN"): \(err?.message ?? "??")\n\n\n\(rawMsg)"
             messageType = .error(err?.type ?? .other)
-            signature = .unknown
+            signature = .error(rawMsg)
```

Comment:
I'd put `.error("processing error")`?. Since the the `text` already contains the issue description (that should be rendered in text)